boards/FSM/Inc/App/configs/App_AcceleratorPedalThresholds.h

```diff
@@ -2,5 +2,10 @@
 
 // TODO: Experimentally determine the encoder values for the primary and
 // secondary APPS when the accelerator is fully pressed #666
-#define PAPPS_ENCODER_FULLY_PRESSED_VALUE 900U
-#define SAPPS_ENCODER_FULLY_PRESSED_VALUE 450U
+#define PAPPS_ENCODER_FULLY_PRESSED_VALUE 2500U
```

Comment:
i think we dont need to pass these into the apps object when we create it. we should move these #defines to App_AcceleratorPedals.c

Reply:
but yeah as mentioned earlier, lets take care of this on another PR